tutor/src/screens/teacher-dashboard/add-assignment-popup.js

```diff
@@ -6,6 +6,26 @@ import Time from '../../models/time';
 import { withRouter } from 'react-router-dom';
 import AddMenu from './add-menu';
 
+const CustomMenu = React.forwardRef(
+  ({ children, className, 'aria-labelledby': labeledBy }, ref) => {
+    return (
+      <div
+        ref={ref}
+        className={className}
+        aria-labelledby={labeledBy}
+      >
+        {children}
+      </div>
+    );
+  },
+);
+
+CustomMenu.propTypes = {
+  'aria-labelledby': PropTypes.string,
+  children: PropTypes.arrayOf(PropTypes.element),
+  className: PropTypes.string,
```

Comment:
You can also use `PropTypes.node` here, that has the advantage of matching anything that can be rendered, while the above will require an array

Reply:
Great tip!